pyproject.toml

```diff
@@ -47,6 +47,7 @@ pytest-cov = {version="^4.0.0", optional=true}
 ruff = {version="==0.2.1", optional=true}
 sphinx = {version="*", optional=true}
 sphinxcontrib-openapi = {version="^0.8.3", optional=true}
+xlrd = "^2.0.1"
```

Comment:
+1

Reply:
It appears that the xlrd dependency is needed if the TLM packet definition file is in .xls format. I changed the format to .xlsx and this dependency is no longer necessary. I will remove it from pyproject.toml.

We do have sample data available for this so that will be next to work on for this new code. Probably in a new PR.